js/apps/admin-ui/src/context/whoami/WhoAmI.tsx

```diff
@@ -22,6 +38,9 @@ export class WhoAmI {
           console.warn("Error(s) loading locale", this.#me?.locale, error);
         }
       });
+      if (RTL_LOCALES.includes(this.#me.locale)) {
+        document.getElementsByTagName("html")[0].setAttribute("dir", "rtl");
```

Comment:
I believe the preferred locale is already known when `index.ftl` is rendered, so you could also handle this at the template level.

Reply:
That's not true atm, we could introduce it in the template though, but why would that be better?

Reply:
Ah, you are right, we only have this for the [account console,](https://github.com/keycloak/keycloak/blob/5fdb572f5f3adaad2d926e09c147e960ca5e8086/js/apps/account-ui/maven-resources/theme/keycloak.v3/account/index.ftl#L2) but not the [admin console](https://github.com/keycloak/keycloak/blob/5fdb572f5f3adaad2d926e09c147e960ca5e8086/js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/index.ftl#L2). The benefit would be that it would be present on initial render, so the browser knows everything about the locale right away.

Reply:
right, but it's almost instant as this will get triggered once the user is logged in

Reply:
True, but it would  be nice to have this working the same as the login and account consoles,